docs/react-integration.md

```diff
@@ -386,16 +407,20 @@ import React from 'react'
 import { queryConnect, Q } from 'cozy-client'
 
 function DumbTodoList(props) {
-  const { data, fetchStatus } = props.todos
+  const { data, fetchStatus, isFetching } = props.todos
   if (fetchStatus !== 'loaded') {
     return <h1>Loading...</h1>
   } else {
     return (
-      <ul>
-        {data.map(todo => (
-          <li key={todo.id}>{todo.label}</li>
-        ))}
-      </ul>
+            <>
+              {isFetching && <h1>Refreshing data...</h1>}
+
+              <ul>
+                {data.map(todo => (
+                        <li key={todo.id}>{todo.label}</li>
+                ))}
+              </ul>
+            </>
     )
   }
 }
```

Comment:
I can't comment on the next commit, the rename one. But all the React stuff should be inside JSX file, not inside JS file.